velox/core/PlanNode.h

```diff
@@ -2144,6 +2144,10 @@ class RowNumberNode : public PlanNode {
     return outputType_;
   }
 
+  bool canSpill(const QueryConfig& queryConfig) const override {
+    return queryConfig.rowNumberSpillEnabled();
```

Comment:
Shall we disable spilling for single partition case?

Reply:
Sure. Updated.